vars/mtaBuild.groovy

```diff
@@ -166,6 +173,48 @@ void call(Map parameters = [:]) {
             $mtaCall
             """
 
+            echo "[INFO] Looping dockerBasedModules."
+
+            def modNames = configuration.dockerBasedModules
+
+	    for( modName in modNames ) {
+               echo "[INFO] modName: '${modName}'."
+               echo "[INFO] MTAR File: '${mtarName}'."
+
+               sh """#!/bin/bash
+               export PATH=./node_modules/.bin:\$PATH
+               echo 'mtarName: '$mtarName
+               rm -rf assemble
+               mkdir -p assemble
+               unzip $mtarName -d assemble
+               export LF=\$(grep -n 'Name: '$modName assemble/META-INF/MANIFEST.MF | cut -d ':' -f 1)
+               echo 'LF:'\$LF
+               export LF1=\$((\$LF-1))
+               echo 'LF1:'\$LF1
+               export LF2=\$((\$LF+5))
+               echo 'LF2:'\$LF2
+               head -n \$LF1 assemble/META-INF/MANIFEST.MF >assemble/META-INF/MAN.MF
+               tail -n +\$LF2 assemble/META-INF/MANIFEST.MF >>assemble/META-INF/MAN.MF
+               cat assemble/META-INF/MAN.MF
```

Comment:
Do we really need all this (debug) output?

Reply:
Nope, I am(was) new to groovy and wasn't sure that the vars were getting set properly.